django_saml2_auth/models.py

```diff
@@ -9,10 +9,11 @@ class SamlMetaData(models.Model):
         blank=True,
     )
     created_at = models.DateTimeField(auto_now_add=True)
-
     metadata_contents = models.TextField()
-
     email_domain = models.TextField(unique=True)
 
+    enable_saml = models.BooleanField(blank=True, null=True,default=False)
+    enable_optional_saml = models.BooleanField(blank=True, null=True,default=False)
```

Comment:
```suggestion
    enable_optional_saml = models.BooleanField(blank=True, null=True,default=False, help_text="when False, SAML is forced if it is enabled. When True, SAML is optional when it is enabled.")
```

(this commit won't pass flake8 so don't commit it directly)